### PR TITLE
Fix Write-Message -Level Error

### PR DESCRIPTION
**Pull request by ClaudioESSilva**
This no longer exists need to replace by the Stop-Function

This also fix the issue #2754

### CODE REVIEW

File: functions/Remove-DbaDatabaseSafely.ps1 [hunk 1]
```diff
@@ -384,7 +384,7 @@ function Remove-DbaDatabaseSafely {
 				}
 			}
 			catch {
-				Write-Message -Level Error -Message "Restore failed: $($_.Exception)."
+				Stop-Function -Message "Restore failed: $($_.Exception)."
```
Comment (potatoqualitee) [inline functions/Remove-DbaDatabaseSafely.ps1]: Thanks, Claudio! The ErrorRecord actually outputs the text you're looking for. Try this instead `Stop-Function -Message "Restore failed" -ErrorRecord $_ -Target $whateveristhetarget` 

but replace target with whatever you need to. unsure why? Take a look at  `Get-DbatoolsLog | OGV`

File: functions/Remove-DbaOrphanUser.ps1 [hunk 2]
```diff
@@ -330,7 +330,7 @@ function Remove-DbaOrphanUser {
 						$User = $null
 					}
 					catch {
-						Write-Message -Level Error -Message $_
+						Stop-Function -Message $_ -Target $db -Continue
```
Comment (potatoqualitee) [inline functions/Remove-DbaOrphanUser.ps1]: Appending `-ErrorRecord $_` would be good here and removing $_ as the message. You could just say "Failure" if you like

File: functions/Remove-DbaDatabaseSafely.ps1 (hunk 1 above)
Comment (ClaudioESSilva) [reply functions/Remove-DbaDatabaseSafely.ps1]: Thanks @potatoqualitee ! done.

File: functions/Remove-DbaOrphanUser.ps1 (hunk 2 above)
Comment (ClaudioESSilva) [reply functions/Remove-DbaOrphanUser.ps1]: Thanks @potatoqualitee ! done.

Comment (wsmelton) [reply functions/Remove-DbaOrphanUser.ps1]: You can actually get that inner exception expanded to a point with Stop-Function by using the `-Exception` parameter. Fred added that months back so you can do:

```
Stop-Function -Message "Whatever failed" -Target <your target variable> -ErrorRecord $_ -Exception $_.Exception -Continue
```

I don't know if you could just pass `$_.Exception.InnerException` as well but not sure how that effects it.

**Comment by potatoqualitee** [comment]
merci! 🌻

**Pull request merged**